src/sensor_mpu9250.c

```diff
@@ -55,14 +59,16 @@ enum {
 static struct task_wake mpu9250_wake;
 
 // Reads the fifo byte count from the device.
-uint16_t
+static uint16_t
 get_fifo_status (struct mpu9250 *mp)
 {
-    uint8_t regs[] = {AR_FIFO_COUNT_H};
+    uint8_t reg[] = {AR_FIFO_COUNT_H};
     uint8_t msg[2];
-    i2c_read(mp->i2c->i2c_config, sizeof(regs), regs, 2, msg);
+    i2c_read(mp->i2c->i2c_config, sizeof(reg), reg, sizeof(2), msg);
```

Comment:
`sizeof(2)` is surely a bug.

Reply:
... how embarrassing. I was trying to standardize on sizeof() vs a mix of sizeof() vs. hardcoded values and ended up with something worse. Fixing now.